docker/linux/android/Dockerfile

```diff
@@ -18,10 +18,11 @@ FROM ${FROM_IMAGE:-cobalt-build-base}
 # TODO: b/309157124 - add --no-install-recommends.
 RUN apt update -qqy
 RUN apt -y install wget curl
```

Comment:
Move `curl` to the `apt` command below. We should only have one `apt` install step in the Dockerfile.

(with the exception of the direct installation of the `.deb` file you are adding). That should be in its own step below.

Reply:
Done